File: test/ayush.dev/python/pipelines/TestNew1/code/testnew1/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from testnew1.config.ConfigStore import *
from testnew1.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0

File: regression2/e2epulloriginlag1/1742217910138/pipelines/Pipeline-Automation-PULL_ORIGIN1-1742217862770/code/src/main/scala/io/prophecy/pipelines/pipelineautomationpull_origin11742217862770/graph/Reformat_1.scala
package io.prophecy.pipelines.pipelineautomationpull_origin11742217862770.graph

import io.prophecy.libs._
import io.prophecy.pipelines.pipelineautomationpull_origin11742217862770.functions.PipelineInitCode._
import io.prophecy.pipelines.pipelineautomationpull_origin11742217862770.functions.UDFs._
import io.prophecy.pipelines.pipelineautomationpull_origin11742217862770.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._
object Reformat_1 { def apply(context: Context, in: DataFrame): DataFrame = in }
